kuiper-game/Domain/CelestialBodies/Star.cs

```diff
@@ -0,0 +1,7 @@
+namespace Kuiper.Domain.CelestialBodies
+{
+    public class Star :CelestialBody
```

Comment:
What are the these classes used for that the baseclass/interface doesn't already solve?

Reply:
It was an attempt to have actual types on the entities, instead of an enum indicating what they are.
For now, they don't actually do anything, other than identify what type of body it is, but potentially they could.